infrateam_first_responder.md

```diff
@@ -11,47 +36,67 @@ The DLSS infrastructure team is trying out a new rotating role of "first respond
 * Encourage cross-training, since a first-responder will likely have to investigate applications with which they are unfamiliar.
 * Shore up missing and outdated documentation of production code and processes for everyone (ops, devops, PSM, stakeholders, devs, etc), informed by actual attempts to find and use said documentation to investigate production issues.
 
-### "First responder" != "on call"
+### "First Responder" != "On Call"
 
 * The idiosyncratic name of the role is intentional.  It is not the duty of the "first responder" to be on-call outside of the first responder's normal work hours (which may not line up exactly with business hours in Palo Alto).  The first responder rotation is an effort to watch and triage production issues in an intentional and organized fashion, since no engineer was officially assigned this responsibility in the past, and such minding of things was haphazard.
 
-## Duties
+# Duties
 
-### Weekly Dependency Updates
+## Weekly Dependency Updates
 
-#### Ensure dependency update spreadsheet is created
+### Ensure dependency update spreadsheet is created
 
 Instructions are here https://docs.google.com/spreadsheets/d/1LysSAPFsRGt9PteWpVswp74xnPXyLLxADpHRfh69VLQ/edit#gid=0
 
 It *may* be that someone in a time zone further east than Palo Alto has already done this, but it is the first responder's responsibility to ensure this gets done.
 
-#### Ensure Monday dependency updates are completed
+### Ensure Monday dependency updates are completed
 
 It is a team task to complete these updates, but the first responder needs to make sure that all codebases needing updates have updates merged and deployed.  Note that some projects need to have PRs created by hand ("Manually clone and update repos that failed bundle update" on line 6 of instructions). It may be helpful to post in channel how many updates each developer should do, given who is working that day and how many PRs there are.
 
-### Verify / Notify coverage for following week
+#### Gems - ensure no security alerts
+
+There is a sheet/tab in https://docs.google.com/spreadsheets/d/1LysSAPFsRGt9PteWpVswp74xnPXyLLxADpHRfh69VLQ/ with a list of sul-dlss repos with both 'infrastructure' and 'gem' tags.  Note that there is no Gemfile.lock, so a PR can't be created automatically.
+1. Check for security alerts for each gem.
```

Comment:
I don't think we need this procedure. If a gem has a security alert, then the application that uses it will have a security alert.  These are reported to us via email from github if subscribed and we get notifications when trying to deploy via capistrano gemfile audit.  If we find a security alert in an application, and trace it to a gem, we can fix it then.

Reply:
cool.  I will remove this section.